src/main/views/claimant-pension-and-benefits.njk

```diff
@@ -29,15 +47,15 @@
               text: anyContributionsLabel
             },
             {
-              text: anyContributions
+              text: anyContributions | safe
```

Comment:
Is it possible to avoid using `safe` here?

Reply:
That is not vulnerable to html injection because we are not getting that value as a user input. So it is safe to use in this case as I am setting html codes in nunjucks between lines 14-20.

% if userCase.claimantPensionContribution === 'Yes' %}
  {% set anyContributions = yes + ",\n" + weekly + ": £" + userCase.claimantPensionWeeklyContribution %}
{% elif userCase.claimantPensionContribution === 'No' %}
  {% set anyContributions = no %}
{% else %}
  {% set anyContributions = notSure %}
{% endif %}